h2/src/main/org/h2/engine/Mode.java

```diff
@@ -204,6 +206,8 @@
      */
     public HashMap<String, DataType> typeByNameMap = new HashMap<>();
 
+    public HashMap<String, FunctionInfo> functionAliases;
```

Comment:
I think this should rather just be a HashMap<String,String> which is more obvious.

Reply:
Actually we need a `FunctionInfo` somethere to allow generation of correct SQL for such functions.

But `FunctionInfo` should not be placed directly in `Mode`.